extensions/ssh-tunnel-manager/src/screens/FormScreen.tsx

```diff
@@ -0,0 +1,71 @@
+import { Action, ActionPanel, Form, useNavigation } from "@raycast/api";
+import { MutableRefObject } from "react";
+import { initForm } from "../hooks/form";
+import { useCache } from "../hooks/store";
+import { TunnelType, Values } from "../types";
+
+export function FromScreen(props: { onSubmit: (values: Values) => void; shouldEstablish: MutableRefObject<boolean> }) {
+  const { shouldEstablish, onSubmit } = props;
+  const { pop } = useNavigation();
+  const { cachedList } = useCache();
+
+  const { itemProps, handleSubmit } = initForm(cachedList, (values: Values) => {
+    if (!shouldEstablish) return;
+    if (shouldEstablish.current) {
+      shouldEstablish.current = true;
+      onSubmit({ ...values });
+    } else {
+      shouldEstablish.current = false;
+      onSubmit({ ...values });
+    }
+    pop();
+  });
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm title="Create and establish tunnel" onSubmit={handleSubmit} />
+          <Action.SubmitForm
+            title="Create tunnel"
+            onSubmit={(values: Values) => {
+              shouldEstablish.current = false;
+              return handleSubmit(values);
+            }}
+          />
+          <Action
+            title="Tunnel List"
+            onAction={pop}
+            shortcut={{
+              modifiers: ["shift"],
```

Comment:
Can we use something else? `⌘+tab` is used in the form to navigate one field up

Reply:
change shortcut to opt+shift+tab